### PR TITLE
Multiple locations in Map

### PR DESCRIPTION
**Pull request by TheSlimvReal**
see issue: #1601 

the storybook showcases it.

### Visible/Frontend Changes
- [x] Added selection in matching view which allows to select one or more location properties of an entity
- [x] Only filtered entities are shown in the map

### Architectural/Backend Changes
- [x] Matching component should work as standalone (currently failing) (already merged -> #1667)

### CODE REVIEW

File: src/app/core/config/config-fix.ts [hunk 1]
```diff
@@ -1115,7 +1118,6 @@ export const defaultJsonConfig = {
       ["address", "address"],
       ["distance", "privateSchool"],
     ],
-    "showMap": ["address", "address"],
```
Comment (sleidig) [inline src/app/core/config/config-fix.ts:1118]: any chance we can keep the config for map properties in this central place? The usual setup integrates the matching in 3 views, each with (almost) same config so it is really tedious to copy and paste.

**Review by sleidig — COMMENTED** [review]
Code looks okay (although the component is - and already has been before - extremely complex).

Visually I think the relation of the dropdowns (and what it means to select from them - i.e. that this is a filter) is not very clear yet. I could imagine that it is more clear if we can integrate the filter into the map view as a filter icon button next to the fullscreen button, which then opens a list of selectable properties. That could be a list of both entity types, I think ("Child: address", "Mentor: work", "Mentor: home", ...). And it might also allow people to filter while viewing the map fullscreen.

File: src/app/core/config/config-fix.ts (hunk 1 above)
Comment (TheSlimvReal) [reply src/app/core/config/config-fix.ts:1118]: Architecture wise it is very annoying to have this separate from the sides. Logically, I definitely belongs to the sides (_which properties of this sides entity should be displayed in the app_). So not having it in the side means a lot of back and forth with indices and searching. I would rather not having to do this. Of course we could just assign it at runtime but that just adds unnecessary complexity. I see the issue with the config but it just seems much more logical this way.

Comment (sleidig) [reply src/app/core/config/config-fix.ts:1123]: we could look into setting default configs for the sides in `"appConfig:matching-entities"` (that is currently the other format but sides are also fixed through that).
But at the moment, maybe let's just keep it as you did and come back to this some time later if we have any ideas to simplify code and config.

**Review by sleidig — APPROVED** [review]

**Pull request merged**